app/Website/src/Http/Resources/Tags/TagResource.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Website\Http\Resources\Tags;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class TagResource extends JsonResource
```

Comment:
BlogPostTagResource

Also, rename all classes from Tag* to BlogPostTag* so that we know what types of tags they are. 

Same goes for the tables. The tags table should be blog_post_tags and the other one blog_post_tag_pivot or something more creative if you can think of a better name.

We may have project idea proposal tags later on.